broadinstitute/pearl
# AR-10 Adding javascript CI

Adds a github action for running javascript tests.  Also does a bunch of lint cleanup.  I'm ignoring eslint in some of the template files as they have some unused variable errors while we're waiting for image capability to be restored.

TO TEST:
1. in this PR, confirm that 2 checks were passed
2. locally, run `npm test` and confirm clean pass
